src/main/resources/assets/styles/main.less

```diff
@@ -1,2 +1,2 @@
-@import "graphiql.min.css";
+@import "../../../../../node_modules/graphiql/graphiql.css";
```

Comment:
`@import "graphiql/graphiql.css"` should be enough. Webpack resolves everything in `node_modules` without having to provide the entire path.